schemas/maas-backend/products/provider.json

```diff
@@ -103,7 +106,7 @@
             "minItems": 1
           }
         },
-        "required": ["id", "name", "inputs"],
+        "required": ["id", "name", "type", "userSelectable", "inputs"],
```

Comment:
backwards incompatible change

Reply:
Hmm i don't really want a major bump. I'll remove this for now